### ISSUE TITLE
fail to launch WSL (both 1 and 2) from terminal after update to win 11 Build 22581.ni_release.220318-1623

### ISSUE THREAD

**Issue by Fan-iX** [microsoft/terminal#12754]
### Windows Terminal version

1.12.10393.0

### Windows build number

Win32NT             10.0.22581.0 Microsoft Windows NT 10.0.22581.0

### Other Software

WSL version: 0.56.2.0
kernel version: 5.10.102.1
WSLg version: 1.0.30
MSRDC version: 1.2.2924
Direct3D version: 1.601.0

### Steps to reproduce

0. terminal profile:

```
            {
                "guid": "{07b52e3e-de2c-5db4-bd2d-ba144ed6c273}",
                "name": "Ubuntu-20.04",  // WSL2
                "source": "Windows.Terminal.Wsl",
                "startingDirectory": "./"
            },
            {
                "guid": "{7a4f49f3-6d68-54b0-90c4-fbd31328a0f9}",
                "name": "Ubuntu.20.04",  // WSL1
                "startingDirectory": "./",
                "source": "Windows.Terminal.Wsl"
            }
``` 

1. Start up WSL directly from windows terminal.
2. Start up WSL from `wsl -d Ubuntu-20.04` or `wsl -d Ubuntu.20.04` from PowerShell, or from <kbd>Win</kbd>+<kbd>r</kbd>.

### Expected Behavior

WSL starts.

### Actual Behavior

1. Both WSL1 and WSL2 failed to start up if launched from Windows Terminal:

```
The parameter is incorrect.

[process exited with code 4294967295 (0xffffffff)]
```

2. Both WSL1 and WSL2 started up and worked properly when launched by PowerShell `wsl -d Ubuntu-20.04`, or <kbd>Win</kbd>+<kbd>r</kbd> `wsl -d Ubuntu-20.04`

**Comment by zadjii-msft** [comment]
Looks a bit like #11994. I think this is fixed in 1.13, but maybe not in 1.12, I can't remember.

**Comment by Fan-iX** [comment]
It worked after removing `/` in `"startingDirectory": "./"` parameter.
The problem only seems to occur in 1.12.
Thank you.
